bin/check-a11y.js

```diff
@@ -15,10 +15,18 @@ const impactColor = {
 }
 
 const { stories } = JSON.parse(readFileSync('dist/stories.json', 'utf8'))
+const pkgFilter = process.argv.slice(2).map((pkgPath) => `./${pkgPath.split('/src')[0]}`) ?? []
 
-const storiesList = Object.keys(stories).reduce((acc, curr) => {
-  if ((stories[curr].importPath).startsWith('./packages/components/') && !(stories[curr].id).endsWith('--docs')) {
-    acc.push(stories[curr].id)
+const storiesList = Object.keys(stories).reduce((acc, cur) => {
+  const isComponentStory = (story) => (story.importPath).startsWith('./packages/components/') && !(story.id).endsWith('--docs')
+  const belongsToFilter = (story, filter) => filter.includes(stories[cur].importPath.split('/src')[0])
```

Comment:
💅 , personally, I prefer to prefix values that represent a boolean with "has" or "is", see https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables

suggestion, `isInFilter`? 🤷